### PR TITLE
remove biggy search cache

### PR DESCRIPTION
**Pull request by frateligu**
#### What problem is this solving?

Biggy search cache has hit rate of 3%. It may be useless

#### How should this be manually tested?

Should be tested on production. After deployment, latency and errors should be analyzed.

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [X] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
✔️  | Technical improvements

### CODE REVIEW

File: node/index.ts [hunk 1]
```diff
@@ -50,8 +48,6 @@ export default new Service<Clients, RecorderState, CustomContext>({
       },
       biggySearch: {
         concurrency: 10,
-        memoryCache: biggySearchCache,
-        timeout: SIX_SECONDS_MS,
```
Comment (thalytafabrine) [inline node/index.ts:54]: why did you remove the `timeout`?

Comment (frateligu) [reply node/index.ts:54]: Thought the timeout was related to the cache.

Comment (thalytafabrine) [reply node/index.ts:54]: I think this timeout is for the existing requests on this client, not for the cache, but it would be nice to check.. 🤔

Comment (frateligu) [reply node/index.ts:54]: According to Athos this is the request timeout. I restored the line.

**Review by thalytafabrine — APPROVED** [review]

**Review by hiagolcm — APPROVED** [review]
